aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -269,6 +269,22 @@ at::Tensor& PackedLinearWeight::apply_relu_out(
   return apply_impl<true>(input, output_scale, output_zero_point, output);
 }
 
+at::Tensor PackedLinearWeight:: apply_leaky_relu(
```

Comment:
I feel error out might make more sense because otherwise user would have silent failures, 

also for users who wants to use fbgemm and happens to have linear - leaky_relu pattern in the model, do they have to specify None for leaky_relu?

Reply:
Below this line is the ref path for FBGEMM. I have added ref path for FBGEMM and QNNPACK. Users will go to ref path if they are using FBGEMM or QNNPACK.
Do you mean I should remove these and error out? I think that would be a problem since linear and leaky_relu will be fused regardless of the backend.